src/lib.rs

```diff
@@ -41,7 +41,7 @@ use warp::{Filter, Rejection};
 /// Wrapper around a request data.
 ///
 /// It is the type that holds the request data extracted by the [`extract_request_data_filter`](fn.extract_request_data_filter.html) filter.
-pub type Request = (FullPath, Method, HeaderMap, Bytes);
+pub type Request = (FullPath, Option<String>, Method, HeaderMap, Bytes);
```

Comment:
Can we abstract this in its own type? Or can we use any other type from the libraries we use already?
Something like:
`pub type QueryParameters = Option<String>`

But I'm pretty sure there should be something already for this.